coldfront/core/project/fields.py

```diff
@@ -0,0 +1,25 @@
+from django import forms
+from django.core.exceptions import ValidationError
+
+from coldfront.core.user.models import User
+
+from coldfront_plugin_qumulo.validators import validate_single_ad_user
+
+
+class PrincipalInvestigatorField(forms.CharField):
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+    
+    def to_python(self, value):
+        return User.objects.get_or_create(username=value)[0]
```

Comment:
Note to self: get_or_create returns a tuple, where the first element of the tuple is the object.
user_obj, created = User.objects.get_or_create(username=value)